downloadservice/app.py

```diff
@@ -286,7 +287,33 @@ def get_upstream_session(user=None):
     return session
 
 
-@app.route(url_prefix + '/health')
+def arcproxy():
+    return int(subprocess.check_output(
+                ["arcproxy", "-i", "vomsACvalidityLeft"]
+              ).decode().strip())
+
+
+def arcinfo():
+    arcinfo = subprocess.check_output(["arcinfo", "-l"]).strip()
+    return dict(
+        proxy_validity_left=arcproxy(),
+        free_slots=int(
+            re.search(r"Free slots: ([0-9]*)", arcinfo.decode()).group(1)),
+        total_slots=int(
+            re.search(r"Total slots: ([0-9]*)", arcinfo.decode()).group(1)),
+    )
+
+
+@app.route(url_prefix + "/health/arc")
+def health_arc():
+    try:
+        return jsonify(arcinfo())
+    except Exception as e:
+        logger.exception("arcinfo result is not good")
```

Comment:
```suggestion
        logger.exception(f"arcinfo result is not good: {e}")
```

Reply:
I set auto-merge so your approval dismissed your comment :(
But, in this case, exception is added anyway by the logger (https://docs.python.org/3/library/logging.html#logging.exception) :)